kernel/src/actions/mod.rs

```diff
@@ -137,8 +137,33 @@ impl Protocol {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Schema)]
-pub struct CommitInfo {
-    pub kernel_version: Option<String>,
+struct OperationParameters {
+    operation_parameters: Option<i64>,
```

Comment:
arrow for some reason doesn't like empty fields. will need to revisit this

Reply:
also follow up for this whole hack in general: https://github.com/delta-incubator/delta-kernel-rs/issues/419

Reply:
Do you mean you can't do just:
```rust
struct OperationParameters;
```

It's unfortunate because the schema gets weird this way.

Reply:
yea.. i had tried this and our expressions will end up constructing an empty StructArray in arrow and it doesn't play nicely when there are no fields. I get the error `Error: Arrow(InvalidArgumentError("Incorrect array length for StructArray field \"operationParameters\", expected 1 got 0"))`

let me take another stab at it